csdap_bulk_download/cli.py

```diff
@@ -91,7 +92,7 @@ def cli(
 
     \b
     The Assets CSV must contain a header row with the following columns:
-      - order_id
+      - collection_id
```

Comment:
The CSV search endpoint returns "provider" as the column title.

Reply:
Good catch.  We ambiguously jump between "provider" and "collection_id" throughout our codebases. At the end of the day, this actually _is_ the `collection_id`.  Should we try to adapt other parts of our system to conform to that? I feel like this ambiguity may eventually bite us.

Reply:
Yeah, it almost did. I'll update the stac api to return `collection_id`.